src/include/wire/libevent_server.h

```diff
@@ -228,18 +228,33 @@ struct LibeventServer {
 
   uint64_t port_;           // port number
   size_t max_connections_;  // maximum number of connections
-  
+  // struct event_base *base;  // libevent event_base
+  struct event *evstop;  // libevent stop event
+  struct event *ev_timeout;
+  std::shared_ptr<LibeventThread> master_thread;
+  // std::shared_ptr<LibeventThread> master_thread(
+  //    new LibeventMasterThread(CONNECTION_THREAD_COUNT, base));
+
+ public:
+  bool is_started = false;
+  bool is_closed = false;
+  struct event_base *base;  // libevent event_base
+  static int recent_connfd;
```

Comment:
Thanks. We may try some other methods to control server's start and close so we make these variables public temporarily. We will set them to private later.